lilypad/lilypad-milky-way-reference/run-a-node.md

```diff
@@ -126,7 +126,7 @@ systemd is a system and service manager for Linux operating systems. systemd ope
 To install systemd, open `/etc/systemd/system/bacalhau.service` in your preferred editor:
 
 {% hint style="info" %}
-Hint: `sudo editor /etc/systemd/system/bacalhau.service`
+Hint: `sudo vim /etc/systemd/system/bacalhau.service`
```

Comment:
Interesting, I thought the `.idea` file that you added is meant for `webstorm` but here you go with `vim`?
In any case, I think we should keep `editor` as that is a generic solution, as opposed to trying to force using `vim`.